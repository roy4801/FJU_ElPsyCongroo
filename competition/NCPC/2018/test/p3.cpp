#include <bits/stdc++.h>

using namespace std;

#define TESTC ""
#define PROBLEM "p3"

#define USE_CPPIO() ios_base::sync_with_stdio(0); cin.tie(0)

int main()
{
	#ifdef DBG
	freopen(PROBLEM TESTC ".in", "r", stdin);
	freopen(PROBLEM ".out", "w", stdout);
	#endif
	int cases, len;

	scanf("%d", &cases);

	while(cases-- && scanf("%d ", &len) != EOF)
	{
		bool b[len] = {false};
		char s[10];

		while(getchar() != '\n');

		for(int i = 0; i < len && (~printf("READ %d\n", i)) && scanf("%s", s) != EOF; i++)
		{
			b[i] = (s[0] == 'f' ? false : true);

			if(i && b[i-1] && !b[i])
			{
				printf("OUTPUT %d\n", i-1);
				break;
			}
		}
	}

	return 0;
}