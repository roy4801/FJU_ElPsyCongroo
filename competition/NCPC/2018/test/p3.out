READ 0
READ 1
READ 2
OUTPUT 1
